action.yml

```diff
@@ -1,35 +1,153 @@
+    required: false
+    
+  comment:
+    description: Add a comment to the ticket after the transition
+    required: false
+
+  missing-transition-as-successful:
+    description: Mark as a successful if issue is missing the transition.
     required: false
+    default: 'true'
+      
+  fail-on-transition-failure:
+    description: Fail if some issues failed transitioning.
+    required: false
+    default: 'true'
+    
+  fail-if-issue-not-found:
+    description: Fail if some issues are not found.
+    required: false
+    default: 'true'
+
+  fail-if-jira-inaccessible:
+    description:  Fail if Jira is inaccessible at the moment. Sometimes Jira is done but shouldn't block the pipeline.
+    required: false
+    default: 'false'
+    
+  jira-username:
+    description: The username to login to Jira with in order to perform the transition. Will be ignored if not set.
+    required: true
+    
   jira-password:
-    description: 'The password to login to Jira with in order to perform the transition. Must be set if jira-username is set. If set when jira-username is not set, it will be ignored.'
+    description: The password to login to Jira with in order to perform the transition. Must be set if jira-username is set. If set when jira-username is not set, it will be ignored.
+    required: true
+    
+  create-notice:
+    description: Add notification to runner with details about successful transitioned issues. 
     required: false
+    default: 'true'
+    
+outputs:
+  processed-issues:
+    description: Issues successfully transitioned, skipped or (if enabled) unavailable.
+    value: ${{ steps.transition.outputs.processedIssues }}
+    
+  failed-issues:
+    description: Issues in Jira not successfully processed.
+    value: ${{ steps.transition.outputs.failedIssues }}
+    
+  unavailable-issues:
+    description: Issues missing the specificed transition.
+    value: ${{ steps.transition.outputs.unavailableTransitionIssues }}
+  
+  not-found-issues:
+    description: Issues not found in Jira.
+    value: ${{ steps.transition.outputs.notfoundIssues }}
+    
+  is-successful:
+    description: One or more issues were transitioned successfully.
+    value: ${{ steps.transition.outputs.isSuccessful }}
+    
+  some-identified:
+    description: Some issues were found in Jira.
+    value: ${{ steps.transition.outputs.has_identifiedIssues }}
+    
+  some-unavailable:
+    description: Some issues do not have transition.
+    value: ${{ steps.transition.outputs.has_unavailableTransitionIssues }}
+    
+  some-skipped:
+    description: Some issues skipped when already transitioned or other causes.
+    value: ${{ steps.transition.outputs.has_skippedIssues }}
+    
+  some-not-found:
+    description: Some issues were not found in Jira.
+    value: ${{ steps.transition.outputs.has_notfoundIssues }}
```

Comment:
I don't see these booleans being outputted by the `transition` step. Am I missing something?

Reply:
https://github.com/im-open/transition-jira-tasks-by-query/pull/9/files#diff-6c3eeb54d4ae2fc59fbf7bd83f739e420765709483afd23c50877763731dd5fdR45

Reply:
Yep, I was definitely missing something 😆